website/docs/reference/resource-configs/athena-configs.md

```diff
@@ -0,0 +1,552 @@
+       1 as quantity,
+       100000000 as quantity_big,
+       current_date as my_date
+```
+
+</TabItem>
+
+<TabItem value="update_condition" >
+
+```sql
+{{ config(
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key=['id'],
+        update_condition='target.id > 1',
+        schema='sandbox'
+    )
+}}
+
+{% if is_incremental() %}
+
+select * from (
+    values
+    (1, 'v1-updated')
+    , (2, 'v2-updated')
+) as t (id, value)
+
+{% else %}
+
+select * from (
+    values
+    (-1, 'v-1')
+    , (0, 'v0')
+    , (1, 'v1')
+    , (2, 'v2')
+) as t (id, value)
+
+{% endif %}
+```
+
+</TabItem>
+
+<TabItem value="insert_condition" >
+
+```sql
+{{ config(
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key=['id'],
+        insert_condition='target.status != 0',
+        schema='sandbox'
+    )
+}}
+
+select * from (
+    values
+    (1, 0)
+    , (2, 1)
+) as t (id, status)
+
+```
+
+</TabItem>
+
+</Tabs>
+
+### High availability (HA) table
+
+The current implementation of table materialization can lead to downtime, as the target table is dropped and re-created. For less destructive behavior, you can use the `ha` config on your `table` materialized models. It leverages the table versions feature of the glue catalog, which creates a temporary table and swaps the target table to the location of the temporary table. This materialization is only available for `table_type=hive` and requires using unique locations. For Iceberg, high availability is the default.
+
+By default, the materialization keeps the last 4 table versions,but you can change it by setting `versions_to_keep`.
+
+```sql
+{{ config(
+    materialized='table',
+    ha=true,
+    format='parquet',
+    table_type='hive',
+    partitioned_by=['status'],
+    s3_data_naming='table_unique'
+) }}
+
+select 'a' as user_id,
+       'pi'     as user_name,
+       'active' as status
+union all
+select 'b'        as user_id,
+       'sh'       as user_name,
+       'disabled' as status
+```
+
+
+#### HA known issues
+
+- There could be a little downtime when swapping from a table with partitions to a table without (and the other way around). If higher performance is needed, consider bucketing instead of partitions.
+- By default, Glue "duplicates" the versions internally, so the last two versions of a table point to the same location.
+- It's recommended to set `versions_to_keep` >= 4, as this will avoid having the older location removed.
+
+### Update glue data catalog
+
+Persist resource descriptions as column and relation comments to the glue data catalog, and meta as [glue table properties](https://docs.aws.amazon.com/glue/latest/dg/tables-described.html#table-properties) and [column parameters](https://docs.aws.amazon.com/glue/latest/webapi/API_Column.html). By default, documentation persistence is disabled, but it can be enabled for specific resources or groups of resources as needed.
```

Comment:
```suggestion
You can persist your column and model level descriptions to the Glue Data Catalog as [glue table properties](https://docs.aws.amazon.com/glue/latest/dg/tables-described.html#table-properties) and [column parameters](https://docs.aws.amazon.com/glue/latest/webapi/API_Column.html). To enable this, set the configuration to `true` as shown in the following examples. By default, documentation persistence is disabled, but it can be enabled for specific resources or groups of resources as needed.
```